integrationTests/testProcessorNodeWithMultisigner.go

```diff
@@ -58,6 +55,26 @@ func NewTestProcessorNodeWithCustomNodesCoordinator(
 		cp.KeyGen,
 		0,
 	)
+	singleSigner := &mock.SignerMock{
+		VerifyStub: func(public crypto.PublicKey, msg []byte, sig []byte) error {
+			return nil
+		},
+		SignStub: func(private crypto.PrivateKey, msg []byte) ([]byte, error) {
+			return nil, nil
+		},
+	}
+	keyGen := &mock.KeyGenMock{}
```

Comment:
I would prefer the signing and verification was the real one not mocked, if we need these mocked we already have the other test processor nodes, this was supposed to be, as the name of the file implies, with multisigner(real)

Reply:
created a `CreateNodesWithNodesCoordinatorAndHeaderSigVerifier`